TCAT/Controllers/InformationViewController.swift

```diff
@@ -351,3 +296,106 @@ class InformationViewController: UIViewController, UITableViewDataSource, UITabl
     }
 
 }
+
+// MARK: Table View Data Source
+extension InformationViewController: UITableViewDataSource {
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return content.count
+    }
+
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return content[section].count
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+
+        // Initalize and format cell
+        let cell = tableView.dequeueReusableCell(withIdentifier: Constants.Cells.informationCellIdentifier, for: indexPath)
+        cell.selectionStyle = .none
+        cell.backgroundColor = Colors.white
+        cell.textLabel?.textColor = Colors.primaryText
+        cell.textLabel?.font = .getFont(.regular, size: 14)
+        cell.textLabel?.textAlignment = .center
+
+        // Set cell content
+        cell.textLabel?.text = content[indexPath.section][indexPath.row].name
+
+        // Set custom formatting based on cell
+        switch (indexPath.section, indexPath.row) {
+
+        case (1, 1): // Send Feedback
+            cell.textLabel?.textColor = Colors.tcatBlue
+        default:
+            break
+
+        }
+
+        return cell
+
+    }
+}
+
+// MARK: Table View Delegate
+extension InformationViewController: UITableViewDelegate {
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let selector = content[indexPath.section][indexPath.row].action
+        performSelector(onMainThread: selector, with: nil, waitUntilDone: false)
+    }
+}
+
+extension InformationViewController: MFMailComposeViewControllerDelegate {
+    @objc func sendFeedback() {
+
```

Comment:
should there be a newline here? should there be a newline after declaring an `extension`? @kevinchan159 what are the rules

Reply:
yeah i pref newline at the beginning and end of `extension` bodies